src/test/scala/nl/biopet/utils/UtilsTest.scala

```diff
@@ -18,5 +19,44 @@ class UtilsTest extends BiopetTest {
     sortAnyAny(text2, text) shouldBe true
     sortAnyAny(one, text) shouldBe true
     sortAnyAny(text, one) shouldBe false
+
+    sortAnyAny(one, three) shouldBe true
+    sortAnyAny(three, one) shouldBe false
+  }
+
+  @Test
+  def testTextToSize(): Unit = {
+    textToSize("10") shouldBe 10L
+
+    textToSize("1k") shouldBe 1024L
+    textToSize("1m") shouldBe (1024L * 1024L)
+    textToSize("1g") shouldBe (1024L * 1024L * 1024L)
+
+    textToSize("1K") shouldBe 1024L
+    textToSize("1M") shouldBe (1024L * 1024L)
+    textToSize("1G") shouldBe (1024L * 1024L * 1024L)
+  }
+
+  @Test
+  def testCamelize(): Unit = {
+    camelize("bla_bla") shouldBe "BlaBla"
+  }
+
+  @Test
+  def testCamelizeToWords(): Unit = {
+    camelizeToWords("BlaBla") shouldBe List("Bla", "Bla")
```

Comment:
Following the success of the Bla movies we have decided to spawn a new series: `Bla and friends`